src/main/java/in/projecteka/gateway/SecurityConfiguration.java

```diff
@@ -97,7 +100,8 @@
             PATH_CONSENT_REQUESTS_ON_INIT,
             PATH_PATIENTS_ON_FIND,
             PATH_HEALTH_INFORMATION_HIP_REQUEST,
-            PATH_HEALTH_INFORMATION_CM_ON_REQUEST
+            PATH_HEALTH_INFORMATION_CM_ON_REQUEST,
+            USERS_AUTH_ON_CONFIRM
```

Comment:
I'm just wondering how it's HIU\_HIP API? Isn't it cm supposed to call this?

Reply:
Not sure about this